apps/docs/src/guide/predicates/methods.md

```diff
@@ -0,0 +1,61 @@
+# Interacting With Predicates
+
+The `Predicate` class extends the [`Account`](https://docs.fuel.network/docs/fuels-ts/account/) class, inheriting all its methods. Therefore, there are multiple ways to interact with predicates, but broadly speaking, we can think about three:
+ - `Checking Balances`
+ - `Transactions`
+ - `Transfers`
+
+
+## Checking Balances
+
+### `getBalances`
+
+This will return the balances of the wallet that is being used to interact with the predicate.
+
+<<< @/../../docs-snippets/src/guide/wallets/checking-balances.test.ts#checking-balances-2{ts:line-numbers}
```

Comment:
The snippet is in `wallets/...` and it should be in `predicates/...`.

Reply:
It's being reused for this purpose but it was originally uses by the wallets documentation.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2345/commits/80b13c26713a2e33bb6a64d252796261e83a7f1a

Reply:
Un-resolving this just to ask: What was the resolution?

Reply:
I must have overlooked this comment, my bad. I agree with your suggestion.

Reply:
See [`2a8de44` (#2345)](https://github.com/FuelLabs/fuels-ts/pull/2345/commits/2a8de44dc130234cfff5cc6b455c9ad6f1be188f)